src/nvim/message.c

```diff
@@ -1560,12 +1560,6 @@ int msg_outtrans_len(const char *msgstr, int len, int attr)
     mode_displayed = false;
   }
 
-  // If the string starts with a composing character first draw a space on
-  // which the composing char can be drawn.
-  if (utf_iscomposing(utf_ptr2char(msgstr))) {
-    msg_puts_attr(" ", attr);
-  }
-
```

Comment:
This change seems wrong. The code below doesn't seem to involve `schar_from_buf_first()`, so this is not done in the code below.

Reply:
`msg_puts_len` -> `msg_puts_display` -> `grid_line_puts` -> `utfc_ptr2schar_len` -> `schar_from_buf_first`

All multibyte rendering ends up in `utfc_ptr2schar[_len]` eventually, that's the point of these refactors..

Reply:
Ah yes, my mistake.